src/main/java/net/sf/jabref/gui/PreviewPanel.java

```diff
@@ -422,4 +431,21 @@ public void actionPerformed(ActionEvent e) {
 
     }
 
+    class CopyAction extends AbstractAction {
```

Comment:
Can you name it CopyPreviewAction as we have a lot of other copy actions there as well.